src/Cake.Issues.Tests/IssueBuilderTests.cs

```diff
@@ -2016,5 +2017,80 @@ public void Should_Set_Run(string run)
                 issue.Run.ToString().ShouldBe(run);
             }
         }
+
+        public sealed class TheWithAdditionalInformationMethod
+        {
+            [Fact]
+            public void Should_Set_Additional_Information()
+            {
+                // Given
+                var fixture = new IssueBuilderFixture();
+
+                // When
+                var issue = fixture.IssueBuilder.WithAdditionalInformation("Foo", "FooValue").Create();
+
+                // Then
+                issue.AdditionalInformation.ShouldContain(new KeyValuePair<string, string>("Foo", "FooValue"));
```

Comment:
This comment goes into the same category as my previous one. Here I had to look if you did not make any typos for "Foo" and "FooValue".

Reply:
Should be clearer now.